haaska.py

```diff
@@ -188,8 +193,12 @@ def response_wrapper(ha, payload):
                            'messageId': str(uuid4()),
                            'payloadVersion': '2'}
             try:
-                func(ha, payload)
-                r['payload'] = {'success': True}
+                res = func(ha, payload)
```

Comment:
Reading the docs again, apparently it's no longer necessary to include {'success': True} in the response payload. So maybe change these few lines to:
`r['payload'] = func(ha, payload) or {}`